### PR TITLE
Add tutorial 701: Pipelines deploy

### PR DESCRIPTION
**Pull request by bjaspan**

### CODE REVIEW

**Comment by bjaspan** [comment]
Review requested by @unn and @kieranacquia.

**Comment by bjaspan** [comment]
Note: These instructions won't work until S3 and GitHub come back online.

File: tutorial-701/acquia-pipelines.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: 1.0.0
+
+variables:
+  global:
+    # Paste your n3_key from ~/.acquia/pipelines/credentials on the next line.
```
Comment (unn) [inline tutorial-701/acquia-pipelines.yaml:5]: I think I'd rather ask people to create a new one. Or at least suggest creating a new one in addition.

File: tutorial-701/acquia-pipelines.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      # If you have build instructions, put them here. They are not required.
+      - build_site:
+          script:
+            - echo "Put your build instructions, if any, here."
+
+      # Deploy the build artifact to a Cloud on-demand environment.
+      - deploy:
+          script:
+            - curl -o $PIPELINES_DEPLOY_TOOL "$PIPELINES_DEPLOY_URL" && chmod 755 $PIPELINES_DEPLOY_TOOL
+            - $PIPELINES_DEPLOY_TOOL
```
Comment (unn) [inline tutorial-701/acquia-pipelines.yaml:24]: Is there any documentation for pipelines-deploy? I might, generally as a developer, want to know what this binary does and maybe see some docs for it before I go about running it. And I'm not seeing it in the readme either.

**Review by unn — DISMISSED** [review]

**Pull request merged**